### PR TITLE
add find departments consumer 

### PR DESCRIPTION
**Pull request by BelehovEgor**
For task http://outsourcing.nat.tepkom.ru:20080/redmine/issues/412

### CODE REVIEW

**Review by KostinP — APPROVED** [review]

**Review by KostinP — APPROVED** [review]
It's work?

File: test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+{
+    class FindDepartmentConsumerTests
+    {
+        private InMemoryTestHarness _harness;
+        private IRequestClient<IFindDepartmentsRequest> _requestClient;
+        private Mock<IDepartmentRepository> _repositoryMock;
+
+        private List<DbDepartment> _departments;
+
+        private const string _dontExistName = "dontExistName";
```
Comment (djino97) [inline test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs:26]: Const fields should be have other naming

**Review by djino97 — APPROVED** [review]

File: src/CompanyService/appsettings.json [hunk 2]
```diff
@@ -24,7 +24,9 @@
 
     // CompanyService receiving endpoints
     "GetUserPositionEndpoint": "CompanyService_GetUserPosition",
-    "GetDepartmentEndpoint": "CompanyService_GetDepartment"
+    "GetDepartmentEndpoint": "CompanyService_GetDepartment",
```
[the remark this answers is not in the document]
Comment (BelehovEgor) [reply src/CompanyService/appsettings.json:27]: it is not mine

File: src/CompanyService.Models.Broker/Responses/IGetDepartmentsNamesResponse.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.Broker.Responses
+{
+    public interface IGetDepartmentsNamesResponse
+    {
+        IDictionary<Guid, string> IdNamePairs { get; }
```
[the remark this answers is not in the document]
Comment (BelehovEgor) [reply src/CompanyService.Models.Broker/Responses/IGetDepartmentsNamesResponse.cs:8]: your name sounds like it is a List, not the dictionary

File: test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs (hunk 1 above)
Comment (BelehovEgor) [reply test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs:26]: how?

File: src/CompanyService.Models.Broker/Requests/IGetDepartmentsNamesRequest.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+using LT.DigitalOffice.Kernel.Attributes;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.Broker.Requests
+{
+    [AutoInjectRequest(nameof(RabbitMqConfig.GetDepartmentsNamesEndpoint))]
+    public interface IGetDepartmentsNamesRequest
+    {
+        IList<Guid> Ids { get; }
```
Comment (KostinP) [inline src/CompanyService.Models.Broker/Requests/IGetDepartmentsNamesRequest.cs:11]: I think it's better "DepartmentIds"

File: test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs (hunk 1 above)
Comment (TIove) [reply test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs:26]: UPPER_CASE or PascalCase, as i remember somewhere in project we already used UPPER_CASE, like DONT_EXIST_NAME

Comment (BelehovEgor) [reply test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs:26]: I didn't find our rules

**Review by TIove — APPROVED** [review]

File: test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs (hunk 1 above)
Comment (TIove) [reply test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs]: outsourcing.nat.tepkom.ru:20080/redmine/projects/digital-office/wiki/Backend_NameConvention

Comment (TIove) [reply test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs]: In word, use CamelCase

**Review by borovHokage — APPROVED** [review]

**Pull request merged**